<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paivita_Kortin_Tietoja</class>
 <widget class="QDialog" name="Paivita_Kortin_Tietoja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnTakaisin">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Takaisin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pin-numero:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Luottotila (1/0):</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>149</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLuottotila">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTilinumero">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tilinumero:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Jos et halua päivittää tiettyä tietoa, jätä kenttä tyhjäksi</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPaivita">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Päivitä</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kortin id:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leKortinId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>10</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Muokkaa kortin tietoja</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePinVaarin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pin annettu väärin (x kertaa):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIdAsiakas">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Asiakas id:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
